docs/react-components-structure.md

```diff
@@ -0,0 +1,161 @@
+---
+id: react-components-structure
+title: React components structure
+---
+
+In Front Commerce we have seperated our components in two categories: the **UI**
+components available in the `ui` folder, and the **Business** components
+available in the `modules` and `pages` folders.
+
+```
+src/
+├── ...
+└── web/
+    └── theme/
+        ├── ui/
+        |   ├── atoms/
+        |   ├── molecules/
+        |   ├── organisms/
+        |   └── templates/
+        ├── modules/
+        └── pages/
+```
+
+But what does that mean? Where should we put our components?
+
+## TL;DR
```

Comment:
At some point, I think parts of it may be better as a blog post and this page may be simpler without the need for this TL;DR part.

All the history could be read in the blog post.

Reply:
Actually I tried and it was really hard.  
I'd request some help to do so.